client/src/components/ReviewFilters/FilteredReviewsStatBreakdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { FilteredReviewsStatContainer, FilteredReviewsStatNormalText, FilteredReviewsStatBoldText, FilteredReviewsStatRatingGroup } from '../../styled';
+
+const FilteredReviewsStatBreakdown = ({ reviewTypeFilter, filteredReviewStats: { totalReviews, ratingGroupInfo: { type, ratingGroup } } }) => {
+  if (!reviewTypeFilter) {
```

Comment:
the ternary is always a clean choice in these situations:
![Screen Shot 2021-02-17 at 12 24 00 AM](https://user-images.githubusercontent.com/66852739/108160154-8a34b180-70b6-11eb-995e-4c780b3b7e24.png)

Reply:
this way you only have one return statement with conditional rendering